contrib/Shopify/sarama/option.go

```diff
@@ -26,7 +33,8 @@ type Option func(cfg *config)
 // WithServiceName sets the given service name for the intercepted client.
 func WithServiceName(name string) Option {
 	return func(cfg *config) {
-		cfg.serviceName = name
+		cfg.consumerServiceName = name
+		cfg.producerServiceName = name
```

Comment:
Okay got it - this is the new change we discussed.

I'm wondering if we want both consumer and producer to be under the same service. It might make sense for queue-based applications to consider consumption/production as operations under that one service.

Reply:
@brettlangdon What do you think?